README.md

```diff
@@ -56,8 +56,38 @@ The above workflow (when manually triggered) will
 ![image](https://user-images.githubusercontent.com/2510597/140506212-4938e44d-0662-4dc5-9fb1-c3f59fe075a6.png)
 
 - When you merge this PR, it will request an Npm OTP from Optic. (If you close the PR, nothing will happen)
-- Upon successful retrieval of the OTP, it will publish the package to Npm.
+- (Optional) You can define Npm and Optic tokens in Github secrets for each user that will receive the otp. This is required only in case you want to publish to Npm.
```

Comment:
Please always use correct casing! (github and otp which is an acronym)

```suggestion
- (Optional) You can define Npm and Optic tokens in GitHub secrets for each user that will receive the OTP. This is required only in case you want to publish to Npm.
```